gcsio/src/main/java/com/google/cloud/hadoop/gcsio/GoogleCloudStorageOptions.java

```diff
@@ -71,13 +71,16 @@ public static Builder builder() {
         .setStorageServicePath(Storage.DEFAULT_SERVICE_PATH)
         .setTraceLogEnabled(false)
         .setTrafficDirectorEnabled(true)
-        .setWriteChannelOptions(AsyncWriteChannelOptions.DEFAULT);
+        .setWriteChannelOptions(AsyncWriteChannelOptions.DEFAULT)
+        .setHnBucketRenameEnabled(false);
```

Comment:
Shouldn't this be used in `GCSImpl.renameHnFolder`?

Reply:
It is used in gcsio/src/main/java/com/google/cloud/hadoop/gcsio/GoogleCloudStorageFileSystemImpl.java

Reply:
Whcih is fine. Given GCSImpl is also a public class does it make sense to have a check in there as well?

Reply:
There is no change in the existing public API. This is a new API. The caller should be doing the check before calling. If the bucket is no HN enabled, they will get an error.